pom.xml

```diff
@@ -719,8 +719,11 @@ limitations under the License.
           <archive>
             <manifestEntries>
               <Build-Date>${project.build.outputTimestamp}</Build-Date>
-              <Build-Version>${project.version}</Build-Version>
-              <Build-Jdk>${java.version}</Build-Jdk>
+                <Build-Version>${project.version}</Build-Version>
+                <Build-Jdk>${java.version}</Build-Jdk>
+                <Specification-Title>Apache Baremaps CSV (incubating)</Specification-Title>
```

Comment:
@rohitYaduvanshi Thanks for this PR. I guess we should probably add a <name> tag in all sub modules and use it to generate the Specification-Title. From what I understand, with this solution, all the generated jars will have the same specification title.

<Specification-Title>${project.name} (incubating)</Specification-Title>

Reply:
@rohitYaduvanshi I merged your commits in #917 as the plan is to start a release process later this week. Thanks a lot for the modification.